### PR TITLE
BZ-1886431 - Remove ignition-generator leftovers

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

**Comment by filanov** [comment]
@ronniel1 what about removing this from the swagger? `ignition_generator_version`

**Comment by ronniel1** [comment]
We'll remove this from swagger in a different PR, after it will be removed from the UI

File: internal/cluster/cluster.go [hunk 1]
```diff
@@ -365,11 +364,6 @@ func (m *Manager) VerifyClusterUpdatability(c *common.Cluster) (err error) {
 	return err
 }
 
-func (m *Manager) SetGeneratorVersion(c *common.Cluster, version string, db *gorm.DB) error {
-	return db.Model(&common.Cluster{}).Where("id = ?", c.ID.String()).
-		Update("ignition_generator_version", version).Error
```
Comment (yevgeny-shnaidman) [inline internal/cluster/cluster.go:370]: don't you want to remove ignition_generator_version from cluster definition in cluster also?

Comment (ronniel1) [reply internal/cluster/cluster.go:370]: if you mean from swagger, see my answer to @filanov above

**Review by filanov — APPROVED** [review]

**Pull request merged**